README.md

```diff
@@ -1,14 +1,615 @@
 # Ably Chat SDK for Android
 
-This is the repository for the Android version of the Ably Chat SDK. We aim to build the same functionality that’s available in the [JavaScript SDK](https://github.com/ably/ably-chat-js).
+<p style="text-align: left">
+    <img src="https://img.shields.io/badge/development_status-Private_Beta-ab7df8" alt="Development status"   />
+    <img src="https://badgen.net/github/license/3scale/saas-operator" alt="License" />
+</p>
+
+Ably Chat is a set of purpose-built APIs for a host of chat features enabling you to create 1:1, 1:Many, Many:1 and Many:Many chat rooms for
+any scale. It is designed to meet a wide range of chat use cases, such as livestreams, in-game communication, customer support, or social
+interactions in SaaS products. Built on [Ably's](https://ably.com/) core service, it abstracts complex details to enable efficient chat
+architectures.
 
 > [!IMPORTANT]
-> This SDK is currently in the early stages of development and is not ready to be used.
+> This SDK is currently under development. If you are interested in being an early adopter and providing feedback then you
+> can [sign up to the private beta](https://forms.gle/vB2kXhCXrTQpzHLu5) and are welcome
+> to [provide us with feedback](https://forms.gle/mBw9M53NYuCBLFpMA). Coming soon: chat moderation, editing and deleting messages.
+
+Get started using the [📚 documentation](https://ably.com/docs/products/chat).
+
+![Ably Chat Header](/images/ably-chat-github-header.png)
 
 ## Supported Platforms
 
-- Android 24 and above
+This SDK works on Android 7.0+ (API level 24+) and Java 8+.
+
+## Supported chat features
+
+This project is under development so we will be incrementally adding new features. At this stage, you'll find APIs for the following chat
+features:
+
+- Chat rooms for 1:1, 1:many, many:1 and many:many participation.
+- Sending and receiving chat messages.
+- Online status aka presence of chat participants.
+- Chat room occupancy, i.e total number of connections and presence members.
+- Typing indicators
+- Room-level reactions (ephemeral at this stage)
+
+If there are other features you'd like us to prioritize, please [let us know](https://forms.gle/mBw9M53NYuCBLFpMA).
+
+## Usage
+
+You will need the following prerequisites:
+
+- An Ably account
+    - You can [sign up](https://ably.com/signup) to the generous free tier.
+- An Ably API key
+    - Use the default or create a new API key in an app within
+      your [Ably account dashboard](https://ably.com/dashboard).
+    - Make sure your API key has the
+      following [capabilities](https://ably.com/docs/auth/capabilities): `publish`, `subscribe`, `presence`, `history` and
+      `channel-metadata`.
+
+[//]: # (## Installation)
```

Comment:
Is this intentionally commented?

Reply:
Yes, I think we should release it in the Maven first